lessons/01_phugoid/01_03_PhugoidFullModel.ipynb

```diff
@@ -729,7 +729,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "* Feng, N. B. et al. _\"On the aerodynamics of paper airplanes\"_, AIAA paper 2009-3958, 27th AIAA Applied Aerodynamics Conference, San Antonio, TX. [PDF](http://www.ntu.edu.sg/home/schluter/pdf/aiaa-2009-3958.pdf)\n",
+    "* Feng, N. B. et al. _\"On the aerodynamics of paper airplanes\"_, AIAA paper 2009-3958, 27th AIAA Applied Aerodynamics Conference, San Antonio, TX. [PDF](http://arc.aiaa.org/doi/abs/10.2514/6.2009-3958)\n",
```

Comment:
Could you replace this with this URL instead: https://www.researchgate.net/publication/261948508_On_the_Aerodynamics_of_Paper_Airplanes

That's a free copy put up by the author.  Thanks for spotting that the link was bad, but we don't want to link to a paywall.